compiler/src/dmd/common/string.d

```diff
@@ -136,8 +136,7 @@ but is guaranteed to follow it.
 version(Windows) wchar[] toWStringz(const(char)[] narrow, ref SmallBuffer!wchar buffer) nothrow
 {
     import core.sys.windows.winnls : CP_ACP, MultiByteToWideChar;
-    // assume filenames encoded in system default Windows ANSI code page
-    enum CodePage = CP_ACP;
+    import dmd.root.filename : CodePage;
```

Comment:
Actually, I don't think dmd/common is allowed to import from dmd/root. dmd/common was created because the backend may not import dmd/root, and now it indirectly does.

Reply:
Oh right, thx. I don't see `toWStringz()` or `extendedPathThen()` being used in the backend, so looks like these should be moved to `dmd.root.filename` then.

Reply:
Ouch, `dmd.common.file` uses `extendedPathThen()`. :(